backend/src/main/resources/graphql/main.graphqls

```diff
@@ -587,13 +592,27 @@ type Mutation {
     patientId: ID!
     dateTested: DateTime
   ): AddTestResultResponse @requiredPermissions(allOf: ["SUBMIT_TEST"])
+  addTestResultMultiplex(
+    deviceId: String!
```

Comment:
any reason why deviceId is a string?

Reply:
Good question... I just copied the existing `addTestResultNew` mutation, which also accepts a string for deviceID, but it probably makes sense to change it to `ID` in the future